src/components/Sidebar.tsx

```diff
@@ -140,44 +150,90 @@ const Sidebar = ({ onClose, ...rest }: SidebarProps) => {
       pos="fixed"
       h="full"
       {...rest}>
-      <Flex 
-        height={topbarHeight}
-        alignItems="center"
-        marginX="8" 
+      <Flex
+        direction="column"
         justifyContent="space-between"
-      >
-        <Box display={{ base: 'none', [sidebarBreakpoint]: 'flex' }}>
-          <NextLink href="http://yuanjian.org" target="_blank">
-            <Image
-              src={yuanjianLogo224x97} 
-              alt="远见教育基金会" 
-              width={112}
-              // Without `priority` we would get a warning from Chrome that this
-              // image "was detected as the Largest Contentful Paint (LCP).
-              // Please add the "priority" property if this image is above the
-              // fold. Read more:
-              // https://nextjs.org/docs/api-reference/next/image#priority"
-              priority
-              />
-            </NextLink>
-        </Box>
-        <CloseButton display={{ base: 'flex', [sidebarBreakpoint]: 'none' }}
-          onClick={onClose} />
-      </Flex>
-      <Box height={{
-        base: 0,
-        [sidebarBreakpoint]: sidebarContentMarginTop - sidebarItemPaddingY,
-      }}/>
+        h="full">
+        <Box>
+          <Flex
+            alignItems="center"
+            marginX="8"
+            marginTop="8"
+            justifyContent="space-between">
+            <Box display={{ base: 'none', [sidebarBreakpoint]: 'flex' }}>
+              <NextLink href="http://yuanjian.org" target="_blank">
+                <Image
+                  src={yuanjianLogo224x97}
+                  alt="远见教育基金会"
+                  width={112}
+                  // Without `priority` we would get a warning from Chrome that this
+                  // image "was detected as the Largest Contentful Paint (LCP).
+                  // Please add the "priority" property if this image is above the
+                  // fold. Read more:
+                  // https://nextjs.org/docs/api-reference/next/image#priority"
+                  priority
+                />
+              </NextLink>
+            </Box>
+            <CloseButton display={{ base: 'flex', [sidebarBreakpoint]: 'none' }} onClick={onClose} />
+          </Flex>
+          <Box height={{
+            base: 0,
+            [sidebarBreakpoint]: 10 - sidebarItemPaddingY,
```

Comment:
avoid hardcoded magic numbers (10 in this case). Use named constants instead (similar to `sidebarContentMarginTop`)